egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -373,6 +373,15 @@ if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
     log "Stage 4: Remove short data: ${data_feats}/org -> ${data_feats}"
 
     for dset in "${train_set}" "${dev_set}" ${eval_sets}; do
+
+        # no need to keep segments & reco2file_and_channel
+        if [ -f ${data_feats}/org/${dset}/segments ]; then
+          rm ${data_feats}/org/${dset}/segments
+        fi
```

Comment:
Sorry, I already added at #1564 about segments, but  I didn't noticed for reco2file_and_channel.
This operation must be done only when feats_type=raw.

Could you send another PR as far common fix? And this problem is also related to tts.sh.

Reply:
please take a look at #1573

Reply:
Remove this changing.